### ISSUE TITLE
Wrong title and "nightly" information for Permissions Policy

### ISSUE THREAD

**Issue by foolip** [w3c/browser-specs#144]
With browser-specs 1.12.0, this is the entry for https://www.w3.org/TR/permissions-policy-1/:
```json
  {
    "url": "https://www.w3.org/TR/permissions-policy-1/",
    "seriesComposition": "full",
    "shortname": "permissions-policy-1",
    "series": {
      "shortname": "permissions-policy",
      "currentSpecification": "permissions-policy-1"
    },
    "seriesVersion": "1",
    "release": {
      "url": "https://www.w3.org/TR/permissions-policy-1/",
      "filename": "Overview.html"
    },
    "nightly": {
      "url": "https://w3c.github.io/webappsec-feature-policy/",
      "repository": "https://github.com/w3c/webappsec-feature-policy",
      "filename": "index.html"
    },
    "title": "Feature Policy",
    "source": "w3c",
    "shortTitle": "Feature Policy"
  }
```

It's an odd mix of the old (Feature Policy) and. new (Permissions Policy). This might be because of the link to https://w3c.github.io/webappsec-permissions-policy/ in https://www.w3.org/TR/permissions-policy-1/, but that link does redirect, so I'm not sure...

**Comment by tidoust** [comment]
Right, the W3C API apparently failed to pick up the title update. I pinged our Webmaster to have the info fixed. URL of the ED should get updated as well although ideally that would be done in the spec itself.

I note that the "This Version" link in the ED actually still uses the former URL. I prepared https://github.com/w3c/webappsec-permissions-policy/pull/400 accordingly.

**Comment by tidoust** [comment]
Fixed in https://github.com/w3c/browser-specs/commit/a658b91af99b7eba28e925a3e718ae7b53e58927 and released in [v1.13.0](https://github.com/w3c/browser-specs/releases/tag/1.13.0)
